packages/frontend/store/index.ts

```diff
@@ -11,14 +11,20 @@ import { composeWithDevTools } from 'redux-devtools-extension';
 
 import userOptionsReducer from './userOptions/reducers';
 import { IUserOptionsState } from './userOptions/types';
+import windowPropsReducer from './system/reducers'
+import { IWindowPropsState } from './system/types';
 
 import rootSaga from './userOptions/sagas';
 
 // Combine reducers and generate the type definition of the AppState
 // https://github.com/reduxjs/redux/pull/3679
 // ^explanation for combineReducers<{}>
-const rootReducer = combineReducers<{ userOptions: IUserOptionsState }>({
+const rootReducer = combineReducers<{
+  userOptions: IUserOptionsState;
+  windowProps: IWindowPropsState;
```

Comment:
Lets call 'windowProps' as just 'system'. Inside of IWindowPropsState, lets just have a single property `mobileView: false` or something.

Reply:
Like this?
```
export interface IWindowPropsState {
  windowProps: WindowProps;
  mobileView: boolean;
}
```

Reply:
Not quite... Something like this:  
```
const rootReducer = combineReducers<{
  userOptions: IUserOptionsState;
  system: ISystemState;

```
and
```
export interface ISystemState {
  mobileView: boolean;
}
```